src/main/java/test/TestMaksym/TestNewUser.java

```diff
@@ -1,57 +1,55 @@
+package main.java.test.TestMaksym;
+
+import io.qameta.allure.Description;
+import io.qameta.allure.Severity;
+import io.qameta.allure.SeverityLevel;
+import main.java.data.User;
+import main.java.data.UserRepository;
+import main.java.pages.login.LoginPage;
+import main.java.pages.main.MainPage;
+import main.java.pages.register.RegisterPage;
+import main.java.test.test;
+import main.java.tools.OurWebDriver;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+@Listeners (test.class)
+
+public class TestNewUser extends test {
+  
+  @Severity(SeverityLevel.CRITICAL)
+  @Description("Verify creation of the new user account")
+	@Test
+	public void testCreateNewUser() throws InterruptedException {
+
+		UserRepository userRepository = new UserRepository();
+
+		MainPage mainPageUserUnregistered = new MainPage();
+		Thread.sleep(500); // for demonstration purposes only
+
+		// click at sign in button to get new LoginPage
+		LoginPage loginPage = mainPageUserUnregistered.clickSignIn();
+		Thread.sleep(500); // for demonstration purposes only
+
+		// click at link "No account? Create one here" to get new RegisterPage
```

Comment:
love the comments
you shouls also add them as @step annotations

Reply:
same as my comment above